packages/grafana-data/src/geo/layer.ts

```diff
@@ -2,74 +2,32 @@ import { Map as OpenLayersMap } from 'ol';
 import BaseLayer from 'ol/layer/Base';
 import { ReactNode } from 'react';
 
+import {
+  FrameGeometrySourceMode,
+  FrameGeometrySource as SchemaFrameGeometrySource,
+  MapLayerOptions as SchemaMapLayerOptions,
+} from '@grafana/schema';
+
 import { EventBus } from '../events';
 import { GrafanaTheme2 } from '../themes';
-import { MatcherConfig, PanelData } from '../types';
+import { PanelData } from '../types';
 import { PanelOptionsEditorBuilder } from '../utils';
 import { RegistryItemWithOptions } from '../utils/Registry';
 
 /**
- * @alpha
+ * @deprecated use the type from schema
  */
-export enum FrameGeometrySourceMode {
-  Auto = 'auto', // Will scan fields and find best match
-  Geohash = 'geohash',
-  Coords = 'coords', // lon field, lat field
-  Lookup = 'lookup', // keys > location
-  // H3 = 'h3',
-  // WKT = 'wkt,
-  // geojson? geometry text
-}
+export { FrameGeometrySourceMode };
 
 /**
- * @alpha
+ * @deprecated use the type from schema
  */
-export interface FrameGeometrySource {
-  mode: FrameGeometrySourceMode;
-
-  // Field mappings
-  geohash?: string;
-  latitude?: string;
-  longitude?: string;
-  h3?: string;
-  wkt?: string;
-  lookup?: string;
-
-  // Path to Gazetteer
-  gazetteer?: string;
-}
+export interface FrameGeometrySource extends SchemaFrameGeometrySource {}
```

Comment:
leviate is not happy with these exports, but not sure how to do it better

Reply:
What will end up in the definition files when we are doing this? The only "actual" breaking change seems to be the `id` on the MatcherConfig that has been removed.

Reply:
Can we do a direct re-export instead? `export type { FrameGeometrySource } from '@grafana/schema';` Tried locally and seems to work, except for `FrameGeometrySourceMode` because it's used as a value, but that one is already re-exported as a value.